_episodes/13-scheduler.md

```diff
@@ -126,6 +137,103 @@ hostname
 ```
 {: .output}
 
+Let's try running it:
+
+```
+$ example-job.sh 
+```
+{: .language-bash}
+
+```
+bash: example-job.sh: command not found...
+```
+{: .error}
+
+Strangely enough, Bash can't find our script. As it turns out, Bash will only
+look in certain directories for scripts to run. To run anything else, we need
+to tell Bash exactly where to look. To run a script that we wrote ourselves, we
+need to tell Bash where it is explicitly, so it doesn't go searching. We could
+do this one of two ways: either with the absolute path 
+`{{ site.workshop_host_homedir }}/yourUserName/example-job.sh` (equivalently,
+`~/example-job.sh`), or with the relative path `./example-job.sh`.
```

Comment:
worth explaining why the `./` is required even when the script is located in the current working directory, as it is here?